include/verilated.cpp

```diff
@@ -45,6 +45,7 @@
 //     and DPI libraries are not needed there.
 //=========================================================================
 
+#include "verilated.h"
```

Comment:
```suggestion
#include "verilated.h"

```

And remove the include verilatedos/verilated_config from below.